install/kubernetes/cilium/values.yaml

```diff
@@ -219,7 +219,10 @@ global:
     # natMax is the maximum number of entries for the NAT table
     natMax: 524288
 
-    # montiorAggregation is the level of aggregation for datapath trace events
+    # mapDynamicSize enables dynamic sizing of BPF maps based on available system memory
+    mapDynamicSize: false
```

Comment:
I think we should enable this by default. The purpose of this is to consume less memory by default. We can keep it disabled for existing deployments and enable it for new deployment by enabling it via the helm values here.

Reply:
Would you enable it by default in the helm chart only or also for the command line option?

Reply:
Not addressed?

Reply:
I addressed it in my local branch but didn't push yet due to the issues I found regarding the correct ratios for the individual maps.

Reply:
Now addressed and set to a ratio of `0.03` (3%) by default which roughly corresponds to the default map sizes on a node with 4 GiB of memory.

This only affects the helm value, not the default value for the command line option (which will set a ratio of `0.0`, i.e. disable the dynamic sizing.